src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -176,8 +176,8 @@ protected virtual Task<TResponse> HandleDelegatedResponseAsync(TResponse delegat
         return remappedResponse;
     }
 
-    public TextDocumentIdentifier GetTextDocumentIdentifier(TRequest request)
+    public VSTextDocumentIdentifier GetTextDocumentIdentifier(TRequest request)
```

Comment:
As above, because of the converter, I don't think we need to change any signatures from `TextDocumentIdentifier` to `VSTextDocumentIdentifier`. It should just be automatically handled. We'd still need to cast of course if anything does need to access the ProjectContext, but I think we don't need any signature changes.